devolutions-gateway/openapi/gateway-api.yaml

```diff
@@ -144,6 +144,34 @@ paths:
       security:
       - scope_token:
         - gateway.heartbeat.read
+  /jet/jrec/delete/{id}:
+    delete:
+      tags:
+      - Jrec
+      summary: Lists all recordings stored on this instance
```

Comment:
It is probably a copy/paste. The endpoint will be used to delete I guess.

Reply:
Good catch. Fixed in https://github.com/Devolutions/devolutions-gateway/pull/1077